Dockerfile

```diff
@@ -55,6 +55,10 @@ ENV LIBMESH_TAG='v1.6.0'
 ENV LIBMESH_REPO='https://github.com/libMesh/libmesh'
 ENV LIBMESH_INSTALL_DIR=$HOME/LIBMESH
 
+# NJOY variables
+ENV NJOY_TAG='2016.65'
```

Comment:
I don't see a good reason to pin the version of NJOY actually. We don't rely on any specific API so there's no risk of breaking our builds like there is for the other things that we are pinning. I'd rather this just grab the latest release version, which is what happens by default when cloning.

Reply:
Now reverted to use the latest release for NJOY.